apps/dashboard/config/application.rb

```diff
@@ -35,6 +35,12 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    config.i18n.default_locale = :en
+    config.i18n.available_locales = %i(en zh-CN)
```

Comment:
I'm not sure if we can specify available locales given some folks may have provided their own at their site without pushing it back to us.